packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -315,8 +317,33 @@ void main() {
       'extendedIconLabelSpacing: 12.0',
       'extendedPadding: EdgeInsetsDirectional(7.0, 0.0, 8.0, 0.0)',
       'extendedTextStyle: TextStyle(inherit: true, letterSpacing: 2.0)',
+      'mouseCursor: MaterialStateMouseCursor(clickable)',
     ]);
   });
+
+  testWidgets('The mouse cursor is themeable', (WidgetTester tester) async {
```

Comment:
I suppose this would be more fitting, wdyt?
```suggestion
  testWidgets('FloatingActionButton.mouseCursor uses FloatingActionButtonThemeData.mouseCursor when specified.', (WidgetTester tester) async {
```